packages/adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -54,54 +54,200 @@ class MyHomePage extends StatelessWidget {
           icon: Icon(Icons.video_call_outlined, color: Colors.black)),
     ];
 
-    // AdaptiveLayout has a number of slots that take SlotLayouts and these
-    // SlotLayouts' configs take maps of Breakpoints to SlotLayoutConfigs.
+    Widget trailingNavRail = Column(
+      children: [
+        const Divider(
+          color: Colors.black,
+        ),
+        const SizedBox(
+          height: 10,
+        ),
```

Comment:
This applies to a lot of the code here but there's not much need to have the code this rigorously indented. If you remove the comma from the last property entry of a Widget and run the autoformatter, it should fix a lot of these indentations. For example, it would change
```
const Divider(
     color: Colors.black,
)
```
to `const Divider(color: Colors.black)`
once you remove the comma after Colors.black.